test/taxonomy/genbank_to_genome/integration_test.py

```diff
@@ -0,0 +1,125 @@
+import unittest
+import os
+import time
+from uuid import uuid4
+from configparser import ConfigParser
+
+from GenomeFileUtil.GenomeFileUtilServer import MethodContext
+from GenomeFileUtil.GenomeFileUtilImpl import GenomeFileUtil
+from installed_clients.WorkspaceClient import Workspace
+
+_DATA_PATH = "/kb/module/test/data"
+
+
+class IntegrationTest(unittest.TestCase):
+    """
+    Basic integration tests on a small genome.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        token = os.environ.get('KB_AUTH_TOKEN', None)
+        # WARNING: don't call any logging methods on the context object,
+        # it'll result in a NoneType error
+        cls.ctx = MethodContext(None)
+        cls.ctx.update({'token': token,
+                        'provenance': [
+                            {'service': 'GenomeFileUtil',
+                             'method': 'please_never_use_it_in_production',
+                             'method_params': []
+                             }],
+                        'authenticated': 1})
+        config_file = os.environ.get('KB_DEPLOYMENT_CONFIG', None)
+        cls.cfg = {}
+        config = ConfigParser()
+        config.read(config_file)  # type: ignore
+        for nameval in config.items('GenomeFileUtil'):
+            cls.cfg[nameval[0]] = nameval[1]
+        cls.wsURL = cls.cfg['workspace-url']
+        cls.ws = Workspace(cls.wsURL, token=token)
+        cls.gfu = GenomeFileUtil(cls.cfg)
+        # create one WS for all tests
+        suffix = int(time.time() * 1000)
+        cls.ws_name = "test_GenomeAnnotationAPI_" + str(suffix)
+        ws_info = cls.ws.create_workspace({'workspace': cls.ws_name})
+        cls.ws_id = ws_info[0]
+
+    def test_genbank_to_genome_ok(self):
+        """
+        Test the `genbank_to_genome` method
+        Taxonomy ID should be pulled from the genbank.
+        """
+        result = self.gfu.genbank_to_genome(self.ctx, {
+            'workspace_name': self.ws_name,
+            'generate_ids_if_needed': 'true',  # why is this a string
```

Comment:
ouch

Reply:
I think it's pretty silly that sdk apps can't have booleans in the json and instead use 0 and 1. Now using a string for a boolean is extra goofy

Reply:
It's because Perl doesn't have a boolean type and the evolution of the SDK started from Perl.